tekton/README.md

```diff
@@ -113,23 +92,23 @@ To use [`tkn`](https://github.com/tektoncd/cli) to run the `publish-tekton-pipel
    kubectl apply -f tekton/resources.yaml
 
    # Change the environment variable to the version you would like to use.
-   # Be careful: due to #983 it is possible to overwrite previous releases.
-   export VERSION_TAG=vX.Y.Z
-   export GIT_RESOURCE_NAME=tekton-pipelines-git-vX-Y-Z
+   export TEKTON_VERSION=vX.Y.Z
+   export TEKTON_RELEASE_GIT_RESOURCE=tekton-pipelines-git-vX-Y-Z
```

Comment:
This could even be `export TEKTON_RELEASE_GIT_RESOURCE=tekton-pipelines-git-${TEKTON_VERSION}` :stuck_out_tongue_closed_eyes: